main/src/cgeo/geocaching/activity/Progress.java

```diff
@@ -108,4 +108,11 @@ public synchronized int getProgress() {
     public synchronized void setProgressDivider(final int progressDivider) {
         this.progressDivider = progressDivider;
     }
+
+    public void incrementProgressBy(final int increment) {
+        if (isShowing()) {
+            dialog.incrementProgressBy(increment);
```

Comment:
On a second look I think I understand what this Progress wrapper is doing. 
Can I remove this `progressDevider` logic? It is not used (always 1, setter never called) and it is buggy, see `getProgress()`.